docs/theming.md

```diff
@@ -217,18 +217,23 @@ html {
 
 ## Shape
 
-MWC doesn't currently support shape customization.
-
-<!-- 
 Material components are all basically rectangular, but allow for some customization of the corner shape. 
 
 By default, components have a rounded corner. The default radius depends on the component's size 
 (small, medium, or large).
 
+```css
+html {
+  --mdc-shape-small: 4px;
+  --mdc-shape-medium: 12px;
+  --mdc-shape-large: 16px;
+}
+```
+
 MWC doesn't support cut corners.
 
-See also: [Material Design shape](https://material.io/design/shape/applying-shape-to-ui.html#).
--->
+See also: [Material Design type system](https://material.io/design/shape/applying-shape-to-ui.html#shape-scheme)
+
```

Comment:
This should still be "shape" and missing a period. Also remove the extra newline so there's only one empty line between the sentence and the next section.

```suggestion
See also: [Material Design shape](https://material.io/design/shape/applying-shape-to-ui.html#shape-scheme).
```

Reply:
gooood catch

Reply:
resolved in the next commit